cmd/pocket.ts

```diff
@@ -76,14 +76,9 @@ const pocketImport = async () => {
 
   const aggsLimitDate = new Date("2021-09-17");
 
-  console.log(`Date from: ${fromDate}`);
-  console.log(`Date to: ${toDate}`);
-  console.log(`List of days: ${days}`);
-  console.log(`Days difference: ${dateDiff}`);
-
   for (const day of days) {
+    let timeUnitMsg = "on day";
```

Comment:
should've declared this out of the loop but its ok

Reply:
noted for next PR

Reply:
oh yeah. I can take that one